### ISSUE TITLE
`ignore : ["consecutive-duplicates"]` is not working correctly.

### ISSUE THREAD

**Issue by BYODKM** [stylelint/stylelint#1078]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug.

[README](https://github.com/stylelint/stylelint/blob/master/src/rules/declaration-block-no-duplicate-properties/README.md) said when `ignore: ["consecutive-duplicates"]` option  has set, `The following patterns are not considered warnings:`. _But it does._

> Which rule, if any, is this issue related to?

`declaration-block-no-duplicate-properties`

> What CSS is needed to reproduce this issue?

```css
p {
  font-size: 16px;
  font-size: 1rem;
  font-weight: 400;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "declaration-block-no-duplicate-properties": [true, {
      "ignore" : ["consecutive-duplicates"]
    }]
  }
}
```

> Which version of stylelint are you using?

`5.4.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint 'main.css'`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

I don't think so.

> What did you expect to happen?

Same behavior as README.md said.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
Unexpected duplicate property "font-size" (declaration-block-no-duplicate-properties)
```

**Comment by jeddy3** [comment]
@BYODKM Thanks for writing up the issue so well. 

The `ignore: ["consecutive-duplicates"]` option was only added a few days ago and will be included in the [next release](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#head). Which will hopefully be really soon.

This isn't the first time that this has happened though and we might need to work out a way to reduce the confusion. Options include:

- A `next` branch, so that `master` docs reflect the current release.
- Smaller and quicker releases. We were getting better at this, but there's some big (and some breaking) changes in the next release and so we're being extra careful.

@BYODKM In the meantime, [the website](http://stylelint.io/user-guide/rules/declaration-block-no-duplicate-properties/)'s documentation is always aligned with the current release. And so you can use that for now.

**Comment by BYODKM** [comment]
@jeddy3 Thank you for your quick response. Got it! Thanks!

**Comment by davidtheclark** [comment]
@jeddy3 Any objection to a swift release of the next version right now? My article on CSS Tricks might be published any minute now and I was thinking it might be nice to get the breaking change out there beforehand.

**Comment by jeddy3** [comment]
@davidtheclark You just caught me before I head out for a couple of hours. Yep, release as `6.0`.

Can you also update the website please as the docs have changes significantly recently.

Awesome news about the article btw :)

**Comment by davidtheclark** [comment]
Published. But something seems to be wrong with the website --- the generated scripts run an infinite loop. Agh!

**Comment by jeddy3** [comment]
I'll try... give me a mo.

**Comment by jeddy3** [comment]
@davidtheclark OK, all done I think. `stylelint-config-standard` is out for support with `6.0.0`. And the website is updated.

**Comment by davidtheclark** [comment]
Thanks. Any idea what could have gone wrong? Because I think I just followed the simple instructions correctly.

**Comment by jeddy3** [comment]
Perhaps something to do with trying out the `demo` branch?

**Comment by jeddy3** [comment]
I'm going pop out for a couple of hours.  🎉 for the `6.0` release btw! :) A big shout out and thanks to all the contributors involved in this release!

**Comment by evilebottnawi** [comment]
@jeddy3 in front of a lot bugfix :smile:

**Comment by jeddy3** [comment]
@BYODKM Released in `6.0.0`.
